apps/rush-lib/src/cli/utilities/InstallManager.ts

```diff
@@ -422,29 +407,37 @@ export default class InstallManager {
       }
 
       if (shouldOverwrite) {
```

Comment:
>shouldOverwrite [](start = 10, length = 15)

When you test this PR, validate that shouldOverwrite is usually false.  If there was a bug in the above logic, Rush would appear to be working correctly, but with a hidden perf regression.

Reply:
I did! Rush always prints the list of updated tgz files.

Reply:
I always check the list of updated tgz files

---
In reply to: [151821951](https://github.com/Microsoft/web-build-tools/pull/445#discussion_r151821951) [](ancestors = 151821951)